css/markdown-editor.styl

```diff
@@ -62,6 +62,7 @@
 .markdown-editor-help
   margin: 2em
   table
+    margin: auto
```

Comment:
Thoughts on adding in `width: 100%` here too? So it'll align the tables vertically and fill the container